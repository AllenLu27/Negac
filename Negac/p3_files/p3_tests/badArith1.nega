int main()
{
  int a;
  bool b;
  a = b - a;
  return 0;
}
