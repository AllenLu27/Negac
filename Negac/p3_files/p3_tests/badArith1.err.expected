FATAL [5,7]: Arithmetic operator applied to invalid operand
Semantic analysis failed
